rita_common/src/debt_keeper/mod.rs

```diff
@@ -617,6 +680,13 @@ impl DebtKeeper {
     }
 }
 
+/// Saves the debt keeper to disk
+pub fn save_debt_to_disk() {
```

Comment:
This is correct in that it avoids DEBT_DATA being public. 

I would prefer if the 'if needed' part of save if needed was decided up in write_to_disk.rs not only for this but for all the other save functions.

Reply:
The issue here is that the save_if_needed for usage tracker is called in send_usage_update. Should I attempt to modify it there as well in order to make this more modularized?

Reply:
do not call save_if_needed in the send _usage_update function, that's one of the goals of this change is to only call the saving functions from wtire_to_disk.rs